src/Components/UserVotes.jsx

```diff
@@ -0,0 +1,31 @@
+import {useState} from "react"
+import { patchArticleVotesById } from "../api"
+import ThumbsDown from "./icons/ThumbsDown"
+import ThumbsUp from "./icons/ThumbsUp"
+
+function UserVotes({article_id, votes}) {
+
+const [userVote, setUserVotes] = useState(0)
+console.log(article_id)
+
+
+function increaseVotes() {
+    setUserVotes((voteChange) => voteChange + 1)
+    patchArticleVotesById(article_id, 1)
+}
+
```

Comment:
It would be great to see a catch here to give a message to the user if it hasn't worked, for example if offline.